.travis.yml

```diff
@@ -25,7 +25,8 @@ script:
   - rvm use $RUBY_VERSION
   - gem install bundler
   - if [ ! -z "$USE_GCC" ] ; then export CC=gcc-$USE_GCC ; export CXX=g++-$USE_GCC ; fi
-  - ./tools/run_tests/run_tests.py -l $TEST -t -j 16 -c $CONFIG -s 4.0
+  - if [ ! -z "$MAX_JOBS" ] ; then export MAX_JOBS=16 ; fi
```

Comment:
This can be done more simply by defining `MAX_JOBS=16` in the global variables at the top.